src/test/java/edu/hm/hafner/analysis/parser/PylintParserTest.java

```diff
@@ -13,7 +13,7 @@
  * Tests the class {@link PyLintParser}.
  */
 class PylintParserTest extends AbstractParserTest {
-    private static final String ISSUES_FILE = "pyLint.txt";
```

Comment:
Would make sense to support the old format as well, so add as additional test

Reply:
Done